backend/server/src/main/java/ai/verta/modeldb/config/MDBConfig.java

```diff
@@ -27,6 +27,7 @@ public class MDBConfig extends Config {
   @JsonProperty private TelemetryConfig telemetry;
   @JsonProperty private List<MigrationConfig> migrations;
   @JsonProperty protected FutureJdbi jdbi;
+  @JsonProperty private boolean enabledPermissionV2 = false;
```

Comment:
I think this reads better with the name `permissionV2Enabled`

Reply:
Renamed